app/serializers/project_serializer.rb

```diff
@@ -1,6 +1,22 @@
 class ProjectSerializer < ActiveModel::Serializer
   # attributes to be serialized
-  attributes :id, :title, :course_id, :user_ids
+  attributes :id, :title, :course_id, :user_ids, :permissions
   # model association
   has_many :stories
```

Comment:
All of these need to be dependent on the read permission of the child resource (see link in ticket)